src/AppBundle/Model/Group.php

```diff
@@ -175,7 +201,7 @@ public function setType($type)
      */
     public function isOfType($type)
     {
-        Assertion::inArray($type, [self::TYPE_LDAP, self::TYPE_FORMAL, self::TYPE_GROUPHUB]);
+        Assertion::inArray($type, [self::TYPE_FORMAL, self::TYPE_SEMI_FORMAL, self::TYPE_AD_HOC]);
```

Comment:
You could add a more descriptive assertion message here. See the `inArray` function for how the message will be formatted.